AcsfToolsCommands.php

```diff
@@ -128,6 +128,8 @@ public function sitesInfo() {
    *   A quoted, space delimited set of arguments to pass to your drush command.
    * @params $command_options Optional.
    *   A quoted space delimited set of options to pass to your drush command.
+   * @option domain-containing
```

Comment:
domain-pattern seems a better naming from my point of view.

Reply:
agreed but it's a bit confusing as pattern usually is used with regex
I tried to keep it more inline with what it is doing

Reply:
got it, updated
and usage is clear - it is checking for the domain that contains the pattern

Reply:
anyways - it's used mostly by devs 😄